kmir/k-src/mir.md

```diff
@@ -41,7 +41,7 @@ The `#return` rule is triggered by the `return` terminator. We need to give it d
 If we are, then we stop execution and enter the finalization phase. Otherwise, if we're not currently in `main`, we return control to the caller function.
 
 ```k
-  rule <k> #return(FUNCTION_KEY, Unit) => .K ... </k>
+  rule <k> #return(FUNCTION_KEY, Unit) => #halt ... </k>
```

Comment:
This rule should be rewritten using the `token` productions idiom rather than with a side condition on the rule.